src/interfaces/terminal/theme.ts

```diff
@@ -50,7 +50,7 @@ export const UI_SYMBOLS = {
   PROVIDER: '◉', // Provider/brain indicator
   FOLDER: '▣', // Thread/folder indicator
   MESSAGE: '◈', // Message count indicator
-  LIGHTNING: '⚡', // Processing indicator
+  LIGHTNING: '.', // Processing indicator
```

Comment:
Why did it change the LIGHTNING to a . ? :)

Reply:
that was actually *me* as a human. I'm fighting against some bugs in inkjs that don't count the width of emoji correctly and corrupt system rendering.